demos/bert_question_answering/python/bert_question_answering.py

```diff
@@ -21,14 +21,47 @@
 
 import numpy as np
 
-from tokens_bert import text_to_tokens, load_vocab_file
+from tokens_bert import text_to_tokens, load_vocab_file, Token
 from html_reader import get_paragraphs
 import grpc
 import numpy as np
 from tensorflow import make_tensor_proto, make_ndarray
 from tensorflow_serving.apis import predict_pb2
 from tensorflow_serving.apis import prediction_service_pb2_grpc
 
+class ConcatenatedParagraph():
+    def __init__(self, text="", tokens=[]):
+        self.text = text
+        self.tokens = tokens
+        self.tokens_length = len(tokens)
+
+    def append_paragraph(self, text, tokens):
+        self.append_tokens(tokens)
+        self.append_text(text)
+
+    def append_text(self, text):
+        self.text += text
+
+    def append_tokens(self, tokens):
+        self.tokens += self.append_offset(tokens, len(self.text), len(self.text))
+        self.tokens_length = len(self.tokens)
+
+    def append_offset(self, tokens, s_offset, e_offset):
+        return list(map(lambda token: Token(token.id, token.start + s_offset, token.end + e_offset), tokens))
+    
+    def get_tokens_id(self):
+        return [token.id for token in self.tokens]
+
+    def get_tokens_se(self):
+        return [(token.start, token.end) for token in self.tokens]
+
+    def get_start(self, token_index):
```

Comment:
Maybe this could be Token object responsibility?
Token.get_start_position() or something like that?

Reply:
We could have get_token method that would return Token on provided index, but the start/end position might be returned by the Token itself.

Reply:
I agree, this functionality should not be the part of this class and should be split just like you suggest.